app/models/authority.rb

```diff
@@ -175,7 +175,7 @@ def median_applications_per_week
   def applications_per_week
     # Sunday is the beginning of the week (and the date returned here)
     # Have to compensate for MySQL which treats Monday as the beginning of the week
-    h = applications.group("CAST(SUBDATE(date_scraped, WEEKDAY(date_scraped) + 1) AS DATE)").count
+    h = by_week_from_sunday(applications, 'date_scraped')
```

Comment:
One thing, I think the method name should make it clear this is going to return a count, rather than groups of records 💯